starter-code/css/main.css

```diff
@@ -0,0 +1,98 @@
+* {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+}
+
+body {
+    width: 900px;

```

Comment:
Setting the width with a hardcoded px value will make this site unresponsive. Try using margin or % instead.